src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Add Web devtools to Flutter from Android on ChromeOS start
+description: Configure your system to develop Flutter web apps on ChromeOS.
+short-title: When you started with Android on ChromeOS
+---
+
+To add Web as a Flutter app target for ChromeOS with Android,
```

Comment:
https://developers.google.com/style/word-list/#web
Also, it's DevTools, not devtools.

```suggestion
To add web as a Flutter app target for ChromeOS with Android,
```